docs/src/components/GatewayIntegrationDiagram.css

```
@@ -0,0 +1,134 @@
+  stroke-width: 1;
+}
+
+[data-theme='dark'] .sts-diagram__backend rect {
+  fill: rgba(255, 255, 255, 0.04);
+  stroke: rgba(255, 255, 255, 0.12);
+}
+
+/* Edges */
+.sts-diagram__edges line,
+.sts-diagram__edges polyline {
+  stroke: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)